docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
+In general, we have done this by sampling correlated *percentiles* at initialization and then keeping those *percentiles*
+static throughout a simulant's life.
+These percentiles, also known as propensities, are applied to different distributions as the simulant changes age group,
+which means that a simulant's risk exposure in one age group and the next are perfectly auto-correlated.
 
-where we have written :math:`e_1` and :math:`e_2` as “standardized”
-exposures, after scaling and shifting so that the TMREL is 0.0 and a one
-unit increment gives the relative increase in disease burden reported by
-GBD.
+For example, a simulant might have their propensities for height and weight sampled at birth from a
+multivariate uniform distribution with correlation.
+If the simulant receives a propensity of 0.85 (85th percentile) in height and 0.80 (80th percentile) in weight,
+they are initially assigned the height and weight values at those percentiles among newborns.
+But when they age into the next age group, they are assigned the values at those same percentiles in their new
+age group, and so on throughout their life.
+In this way, their height and weight can change over time, but the correlation introduced in the initial sampling
+of propensities remains.
 
-With the log-linear assumption, we can calculate the PAF from first
-principles as a double integral (below :math:`p_{e_1,e_2}` denotes the
-joint probability density of exposure level :math:`(e_1,e_2)` in the
-population:
+.. todo::
+  Be more exact about the properties of the propensity correlation approach.
+  I believe that it only exactly maintains the correlation coefficient with some distributional assumptions.
 
-..
-  :raw-latex:`\begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  \end{align*}`
+Risks with effects on the same outcome
+--------------------------------------
 
-.. math::
-  :nowrap:
+When we introduce correlation between two risk factors that have effects on the same outcome, we have to
+be especially careful about double-counting issues.
 
-  \begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2
-  \end{align*}
+The population-attributable fractions (PAFs) must be calculated correctly,
+or the population-level incidence of the outcome will
+not validate to GBD.
```

Comment:
I think we should at context to what we mean by "correctly" here. Here's my outline:

- If we do not model correlation between risks that are truly correlated and use the multiplicative PAF, we will validate to GBD's outcome rate, but we may overestimate burden attributable to these two risks (although it will match GBD's estimate).
- If we model correlation between risks that are truly correlated and use the multiplicative PAF, we may not validate to GBD's outcome rate
- If we model correlation between risks that are truly correlated and use the joint PAF, we will validate to GBD's outcome rate and will have less bias in our estimate of burden attributable to these two risks (although it will be lower than GBD's estimate of attributable burden)

Reply:
Here's how I'm thinking about it:

- There is no reason I can think of to ever do the second option you listed -- we always want to treat correlation and joint PAF as a package deal.
- The choice between the first and third options is about the biases that can be introduced by omitting a real-life risk correlation in our model structure. This gets to what we were talking about yesterday (when/why should you include a risk correlation?) and falls under the TODO I put above about this.

That said, I don't think the word "correctly" is very clear here so I will revise.

Reply:
Agreed on your points here, sounds good.